src/components/game/GuessWordItem.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  Image,
+  Pressable,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import React from 'react';
+import {CSText} from '../core';
+import {SPACING} from '../../constants/spacing';
+import {COLORS} from '../../constants/color';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+interface GuessWordItemProps {
+  image: string;
+  word: string;
+  onPressItem: () => void;
+  onRemoveItem: () => void;
+  isActive: boolean;
+  index: number;
+}
+
+const GuessWordItem = ({
+  image,
+  word,
+  onPressItem,
+  onRemoveItem,
+  isActive,
+  index,
+}: GuessWordItemProps) => {
+  return (
+    <Pressable onPress={onPressItem} style={styles.container}>
+      <Image
+        source={{
+          uri: image,
+        }}
+        style={styles.image}
+      />
+      <View style={[styles.space, isActive && styles.active]}>
+        {word !== '' && isActive && (
+          <TouchableOpacity
+            style={[styles.remove, index % 2 === 0 ? styles.event : styles.odd]}
+            onPress={onRemoveItem}>
+            <Icon name="close" color={COLORS.red} style={styles.iconRemove} />
+          </TouchableOpacity>
+        )}
+        <CSText color="primaryDark" variant="Bungee">
+          {word}
+        </CSText>
+      </View>
+    </Pressable>
+  );
+};
+
+export default GuessWordItem;
+
+const styles = StyleSheet.create({
```

Comment:
Create new file GuessWordItem.styles.ts and move styles to file GuessWordItem.styles.ts